<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Maratis Manager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="EditorTab">
       <attribute name="title">
        <string>Editor</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Projects</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Recent projects:</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QComboBox" name="lastUsedCombo">
                 <property name="editable">
                  <bool>true</bool>
                 </property>
                 <item>
                  <property name="text">
                   <string>Empty</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="openProjectButton">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QPushButton" name="pushButton">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>75</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Open</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>506</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="UpdatesTab">
       <attribute name="title">
        <string>Updates</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Installation</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="1">
              <widget class="QComboBox" name="sourceCombo">
               <item>
                <property name="text">
                 <string>Official Sourcecode via SVN</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Binary release</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Community fork via GIT</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="installButton">
               <property name="text">
                <string>Install</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Update source:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Compiler settings</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Threads:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="threadsBox">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="keepCodeBox">
             <property name="text">
              <string>Keep sourcecode after installation</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="progressLabel">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="outputTab">
       <attribute name="title">
        <string>Editor Output</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>517</width>
             <height>350</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTextEdit" name="consoleOutput">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="lineWidth">
               <number>0</number>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>openProjectButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>openProjectButtonClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>410</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startEditorButtonClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>installButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>installUpdatesButtonClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sourceCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateSourceChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openProjectButtonClick()</slot>
  <slot>startEditorButtonClick()</slot>
  <slot>installUpdatesButtonClick()</slot>
  <slot>updateSourceChanged()</slot>
  <slot>startColorChooser()</slot>
 </slots>
</ui>
